### PR TITLE
React preview display name

### PR DESCRIPTION
**Pull request by samuel-peers**
Associated Issue: #4287

### Summary of Changes

Show the react component display name in the preview

### Screenshots
react_display_name

### CODE REVIEW

File: src/components/Editor/Preview/Popup.css [hunk 1]
```diff
@@ -15,12 +15,18 @@
...
   line-height: 20px;
   border-bottom: 1px solid #cccccc;
   display: flex;
   flex-direction: column;
+  margin-bottom: 5px;
+}
+
+.popover .preview-popup .header-container h3 {
+  margin: 0;
+  margin-bottom: 5px;
```
Comment (jasonLaster) [inline src/components/Editor/Preview/Popup.css]: lets soften the header a bit...
smaller font, normal weight...

Comment (samuel-peers) [reply src/components/Editor/Preview/Popup.css]: Kay kay how's this:
react_display_name_soft

Comment (jasonLaster) [reply src/components/Editor/Preview/Popup.css]: Looks better to me. As a small detail for the screens, can you turn off the feature w/ the bottom bar :) it's not ready to ship yet

Comment (jasonLaster) [reply src/components/Editor/Preview/Popup.css]: also, this is a nit, but can we go drop the border below App? might be nice

Comment (samuel-peers) [reply src/components/Editor/Preview/Popup.css]: > "can you turn off the feature w/ the bottom bar"

Do you mean the bottom bar of the preview?

Comment (jasonLaster) [reply src/components/Editor/Preview/Popup.css]: yeah - it's in launchpad, settings "Add watch expression from preview"

**Comment by samuel-peers** [comment]
preview without header bottom border:
react_display_name_no_bottom_bar

**Comment by samuel-peers** [comment]
Hmm there seems to be a little styling issue when "Add watch expression from preview" is unchecked:
react_display_name_no_bottom_bar_issue

**Pull request merged**